addBag.php

```diff
@@ -19,26 +25,40 @@
         <form method ="post">
             <div>
                 <label>Brand: </label>
-                <input type="text" name="brand" required/><br>
+                <input type="text" name="brand" /><br>
             </div>
             <div>
                 <label>Model: </label>
-                <input type="text" name ="model" required/><br>
+                <input type="text" name ="model" /><br>
             </div>
             <div>
                 <label>Price (£): </label>
-                <input type="text" name ="price" required/><br>
+                <input type="text" name ="price" /><br>
             </div>
             <div>
-                <label>Materials (optional): </label>
-                <input type="text" name ="materials"/><br>
+                <label>Materials: </label>
+                <input type="textarea" name ="materials" /><br>
             </div>
             <div>
                 <label>Image Address: </label>
-                <input type="url" name ="image address" required/><br>
+                <input type="url" name ="image" /><br>
```

Comment:
Same thing here. Consider giving the input an id of "image" and the label a for attribute of "image"